walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Same comment as in PR #2  -  I think this file can be deleted

Reply:
Addressed in 3.5part1